.github/workflows/run_tests.yml

```
@@ -53,22 +53,15 @@ jobs:
         if: ${{ matrix.installation_required }}
         run: sudo xcodes runtimes install "iOS ${{ matrix.sdk }}"
 
-      - name: Build
-        run: |
-          xcodebuild \
-            -scheme "Listable-Package" \
-            -destination "platform=${{ env.TUIST_TEST_PLATFORM }} Simulator,OS=${{ matrix.sdk }},name=${{ env.TUIST_TEST_DEVICE }}" \
-            -sdk iphonesimulator build
-
       - name: Install Tuist dependencies for Development
         run: tuist install --path Development/
 
       - name: Test All
-        run: tuist test ListableDevelopment-Workspace --path Development/ --os ${{ matrix.sdk }} --result-bundle-path TestResult.xcresult
+        run: tuist test ListableDevelopment-Workspace --path Development/ --platform ${{ env.TUIST_TEST_PLATFORM }} --os ${{ matrix.sdk }} --device "${{ env.TUIST_TEST_DEVICE }}" --result-bundle-path TestResult.xcresult
```

The `TUIST_*` env vars are actually used by tuist by default, so you don't need command line args for them. [tuist test docs](https://docs.tuist.dev/en/cli/test)

```suggestion
        run: tuist test ListableDevelopment-Workspace --path Development/ --os ${{ matrix.sdk }} --result-bundle-path TestResult.xcresult
```